extensions/amp-analytics/0.1/crypto-impl.js

```diff
@@ -93,6 +104,20 @@ export class Crypto {
       return result;
     });
   }
+
+  /**
+   * Loads Crypto polyfill library.
+   * @returns {!Promise}
+   * @private
+   */
+  loadPolyfill_() {
+    if (this.polyfillPromise_) {
+      return this.polyfillPromise_;
+    }
+    return this.polyfillPromise_ = extensionsFor(this.win_)
+        .loadExtension('amp-crypto-polyfill')
+        .then(() => getExistingServiceForWindow(this.win_, 'crypto-polyfill'));
```

Comment:
You already lookup window-scoped service, so it should definitely be window-scoped in the polyfill itself.